sdk/data/azcosmos/cosmos_global_endpoint_manager.go

```diff
@@ -0,0 +1,170 @@
+package azcosmos
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"net/url"
+	"sync"
+	"time"
+
+	azlog "github.com/Azure/azure-sdk-for-go/sdk/azcore/log"
+	azruntime "github.com/Azure/azure-sdk-for-go/sdk/azcore/runtime"
+	"github.com/Azure/azure-sdk-for-go/sdk/internal/log"
+)
+
+const DefaultBackgroundRefreshInterval = 5 * time.Minute
```

Comment:
Casing defines accessor in Go. This is an internal/private const, use lower camel case:

```suggestion
const globalEndpointManagerDefaultBackgroundRefreshInterval = 5 * time.Minute
```